docs/manual/scala/guide/cluster/code/docs/home/scaladsl/persistence/ShoppingCart.scala

```diff
@@ -28,7 +28,7 @@ import play.api.libs.json._
 
 import scala.collection.immutable.Seq
 
-object ShoppingCart {
+object ShoppingCartExamples {
```

Comment:
Why was this renamed?

Reply:
By the way, the build is broken because of it.

Reply:
Before, the state was modelled as `State` and the `ShoppingCart` was the wrapper. It was renamed to have a better name for the model, bit also to be able to have an object called `ShoppingCart` under `ShoppingCartExamples`. 

Then we can use the `ShoppingCart` object to declare the `Behavior` and the `EntityTypedKey`.